src/client/roles/RolesService.ts

```diff
@@ -0,0 +1,182 @@
+import { HaysonDict, HDict, HGrid, HMarker, HRef } from 'haystack-core'
+import { ClientServiceConfig } from '../ClientServiceConfig'
+import { fetchVal } from '../fetchVal'
+import { dictsToGrid } from '../../util/hval'
+import { encodeQuery } from '../../util/http'
+import { ReadOptions } from '../RecordService'
+
+/**
+ * a role record
+ */
+export interface Role extends HDict {
+	id?: HRef
+	userRole: HMarker
+}
+
+/**
+ * optional parameters available to readByFilter query
```

Comment:
Extreme total nitpik: sentances should start with a capital letter.

Reply:
Added.